apps/marsha/vars/all/main.yml

```diff
@@ -75,3 +65,32 @@ marsha_postgresql_resources:
   requests:
     cpu: 10m
     memory: 100Mi
+
+# -- Cronjobs
+marsha_deploy_clean_medialive_dev_stack_cronjob: false
```

Comment:
Is it used elsewhere?

Reply:
In the project no, but this is the only redefine by environment